app/views/health_cards/show.html.erb

```diff
@@ -0,0 +1,103 @@
+<%= javascript_pack_tag 'details_pg' %>
+<p id="notice"><%= notice %></p>
+<section class="hero is-link">
+  <div class="hero-body">
+    <p class="title">Health Card Details</p>
+  </div>
+</section>
+
+  <section class="section">
+    <div class="container">
+
+      <div class="card is-fullwidth">
+        <header class="card-header">
+          <p class="card-header-title">Expanded JWS Payload</p>
+          <a class="card-header-icon card-toggle">
+            <i class="fa fa-angle-down"></i>
+          </a>
+        </header>
+          <div class="card-content">
+          <div class="content">
+            <pre style="white-space:pre-wrap;">
+              <p> <%= JSON.pretty_generate(@health_card.to_hash) %> </p>
+            </pre>
+            </div>
+          </div>
+        </div>
+    <br>
+    <br>
+
+      <div class="card is-fullwidth">
+        <header class="card-header">
+          <p class="card-header-title">Minified JWS Payload</p>
+          <a class="card-header-icon card-toggle">
+            <i class="fa fa-angle-down"></i>
+          </a>
+        </header>
+          <div class="card-content">
+          <div class="content">
+            <pre style="white-space:pre-wrap;">
+              <p> <%= @health_card.to_json %> </p>
+            </pre>
+            </div>
+          </div>
+        </div>
+    <br>
+    <br>
+
+    <div class="card is-fullwidth">
```

Comment:
Could we move the FHIR Bundle as the top thing on this page, since this is what is created first? Other than that, it looks like everything else is in order.

Reply:
ah good point I renamed and forgot to move it...moved to the top